File: app/src/main/java/com/coder/ffmpegtest/BaseApplication.java
package com.coder.ffmpegtest;

import android.app.Application;

import com.coder.ffmpeg.jni.FFmpegCommand;
import com.tencent.bugly.crashreport.CrashReport;

/**
 * @author: AnJoiner
 * @datetime: 19-12-20
 */
public class BaseApplication extends Application {
    private static BaseApplication instance;

    @Override
    public void onCreate() {
        super.onCreate();
        instance = this;
        FFmpegCommand.setDebug(true);
        CrashReport.initCrashReport(getApplicationContext(), "d7b0e14940", true);
    }

    public static BaseApplication getInstance() {
        return instance;
    }
}

File: app/src/main/java/com/coder/ffmpegtest/BaseApplication.kt
package com.coder.ffmpegtest

import android.app.Application
import com.coder.ffmpeg.jni.FFmpegCommand
import com.tencent.bugly.crashreport.CrashReport

/**
 * @author: AnJoiner
 * @datetime: 19-12-20
 */
class BaseApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        instance = this
        FFmpegCommand.setDebug(false)
        CrashReport.initCrashReport(applicationContext, "d7b0e14940", true)
    }

    companion object {
        var instance: BaseApplication? = null
            private set
    }
}